.drone.yml

```diff
@@ -0,0 +1,129 @@
+---
+
+kind: pipeline
+type: docker
+name: php-8.0
+
+clone:
+  disable: true
+
+steps:
+  - name: test-php-8.0
+    image: shaarli/drone-ci-base:8.0
+    commands:
+      - git clone --depth=50 --branch=$DRONE_BRANCH https://github.com/nodiscc/Shaarli.git .
```

Comment:
The hard-coded URL might expose some problems on the long run (testing PRs for example), why not let Drone autoclone? https://docs.drone.io/pipeline/docker/syntax/cloning/

The only time I ever needed turning off autocloning was with submodules or deploy token SSH auth with git. 

If this is only for the current migration phase for easier testing outside of the Shaarli repo, please ignore this comment :)

Reply:
> why not let Drone autoclone?

because it always clones as root as far as I know (https://github.com/drone/drone-git/pull/23), then we can't run the test suite as a non-root user (can't write in the working directory), and then if we run it as root, tests that expect unwriteable/unreadable files will fail because root can r/w any file.

I found this workaround a while ago and don't remember the exact details, so we may want to give it another try.

The hardcoded URL is definitely not good though